src/components/SearchBar/components/AddressBook/AddressBook.tsx

```diff
@@ -87,27 +90,34 @@ const AddressBook: React.FC = () => {
     setIsOpen(true);
   }, []);
 
-  const handleAddressAdd = useCallback((newEntry: string) => {
-    console.log("adding new address", newEntry);
-    setAddressBookEntries((prev) => {
-      // first address book entry
-      if (prev === undefined) {
-        return [newEntry];
-      }
-
-      if (prev.includes(newEntry)) {
-        // TODO: improve error handling
-        console.error("address already saved:", newEntry);
-        return prev;
-      }
-      return [...prev, newEntry];
-    });
-  }, []);
+  const handleAddressAdd = useCallback(
+    (newEntry: string) => {
+      setAddressBookEntries((prev) => {
+        // first address book entry
+        if (prev === undefined) {
+          enqueueSnackbar(messageText.addressBook.success, { variant: "success" });
```

Comment:
if you wanted for convenience, you could add some small wrapper functions for 'notifySuccess' and it would automatically include the variant success when it calls enqueueSnackbar(). then an error version, info version, etc could also be done for shortcuts.

definitely a suggestion for convenience and not really required.

Reply:
I tried doing this but i'm putting it off for now because it requires a hook due to `enqueueSnackbar` being a hook. If it were just a utils function I would have already made the convenience function. Will make an issue though.